libraries/provisioning/provisioning.c

```diff
@@ -228,23 +228,25 @@ static uint32_t send_ack_packet(void)
     res = Shared_Data_sendData(&data_to_send, packet_sent_cb);
     if (res != APP_LIB_DATA_SEND_RES_SUCCESS)
     {
-        LOG(LVL_WARNING, "State WAIT_DATA : Error sending ACK (res:%d).", res);
-        return APP_SCHEDULER_SCHEDULE_ASAP;
+        LOG(LVL_WARNING, "State WAIT_DATA : Error sending ACK (res:%u).", res);
```

Comment:
I think setting "m_events.timeout = 1;" here instead would be cleaner.
It should have exactly same behavior.